packages/number-input/src/NumberInput/NumberInput.styles.ts

```diff
@@ -47,17 +56,23 @@ export const unitDisabledStyles: Record<Theme, string> = {
   `,
 };
 
-export const errorMessageWrapperStyles = css`
-  font-size: ${typeScales.body1.fontSize}px;
-  line-height: ${typeScales.body1.lineHeight}px;
+export const errorContainerStyles = css`
+  margin-top: ${spacing[100]}px;
```

Comment:
Since we always show the error container, the margin-top is always there, even if there is no error text. Can we update this so it only shows up when there is text?